micrometer-core/src/main/java/io/micrometer/core/instrument/Timer.java

```diff
@@ -423,7 +424,7 @@ public Builder pauseDetector(PauseDetector pauseDetector) {
         }
 
         @Override
-        public Builder description(String description) {
+        public Builder description(@Nullable String description) {
```

Comment:
Adding `@Nullable` here and the other place in this file was an attempt to fix the `NullTernary` error, but it did not help. I can revert since it isn't really part of enabling errorprone.